src/Context/TqContext.php

```diff
@@ -262,7 +262,11 @@ public function beforeStep(BehatScope\StepScope $scope)
     {
         self::$pageUrl = $this->getCurrentUrl();
         // To allow Drupal use its internal, web-based functionality, such as "arg()" or "current_path()" etc.
-        $_GET['q'] = ltrim(parse_url(static::$pageUrl)['path'], '/');
+        $path = ltrim(parse_url(static::$pageUrl)['path'], '/');
+        if ('' == $path) {
```

Comment:
This special case is from drupal_path_initialize:
https://api.drupal.org/api/drupal/includes!path.inc/function/drupal_path_initialize/7
I guess it's for cases when the path is not defined like the homepage. Not sure if it's applicable in our scenario, needs tests